pandas/core/arrays/string_arrow.py

```diff
@@ -331,6 +333,7 @@ def _maybe_convert_setitem_value(self, value):
 
     def isin(self, values):
         if pa_version_under2p0:
+            fallback_performancewarning(version="2")
```

Comment:
can you check that we are producing this in tests?

Reply:
Testing this now with `tm.assert_produces_warning`